game/assets/jak1/text/game_text_de.gs

```diff
@@ -0,0 +1,189 @@
+(#x1040 "UNTERTITEL EINGESCHALTET")
+(#x1041 "UNTERTITEL AUSGESCHALTET")
+(#x1042 "TEXTSPRACHE")
+
+(#x1050 "MSAA")
+(#x1051 "~DX")
+(#x1052 "2X")
+(#x1053 "4X")
+(#x1054 "8X")
+(#x1055 "16X")
+
+(#x1060 "BILDRATE")
+(#x1061 "60")
+(#x1062 "100")
+(#x1063 "150")
+
+(#x1070 "DETAILLIERUNGSGRAD (HINTERGRUND)")
+(#x1071 "DETAILLIERUNGSGRAD (VORDERGRUND)")
+(#x1072 "MAXIMUM")
+(#x1073 "HOCH")
+(#x1074 "MITTEL")
+(#x1075 "NIEDRIG")
+(#x1076 "MINIMUM")
+(#x1077 "PS2")
+
+(#x1078 "UNTERTITEL")
+(#x1079 "HINWEIS-UNTERTITEL")
+(#x107a "UNTERTITELSPRACHE")
+(#x107b "SPRECHER IN UNTERTITELN ANZEIGEN")
+(#x107c "IMMER")
+(#x107d "NIEMALS")
+(#x107e "OFFSCREEN")
+
+(#x107f "HINWEIS-LOG")
+
+(#x1080 "CHEATS")
+(#x1081 "GEHEIMNISSE")
+(#x1082 "LEVEL AUSWÄHLEN")
+(#x1083 "VARIANTE AUSWÄHLEN")
+(#x1084 "FINALER BOSS")
+(#x1085 "CREDITS")
+(#x1086 "?????")
+(#x1087 "KLAWW")
+(#x1088 "FISCHER-MINISPIEL")
+(#x1089 "HINTERHALT")
+
+(#x1090 "UNENDLICH BLAUES ECO")
+(#x1091 "UNENDLICH ROTES ECO")
+(#x1092 "UNENDLICH GRÜNES ECO")
+(#x1093 "UNENDLICH GELBES ECO")
+(#x1094 "ALTERNATIVER DAXTER")
+(#x1095 "UNVERWUNDBARKEIT")
+(#x1096 "ALLE MUSIKVARIANTEN")
+(#x1097 "TAG-NACHT-ZYKLUS IN ECHTZEIT")
+
+(#x10c0 "MUSIK-PLAYER")
+(#x10c1 "SZENEN-WIEDERHOLUNG")
+(#x10c2 "CREDITS ABSPIELEN")
+(#x10c3 "ALBUM")
+
+(#x10d0 "STANDARD")
+(#x10d1 "UNBENUTZT")
+(#x10d2 "SAMOS")
+(#x10d3 "HÜTTE DES GRÜNEN WEISEN")
+(#x10d4 "VOGEL-BEOBACHTERIN")
+(#x10d5 "FARMER")
+(#x10d6 "KEIRA")
+(#x10d7 "BÜRGERMEISTER")
+(#x10d8 "BILDHAUER")
+(#x10d9 "JAKS ONKEL")
+(#x10da "HAFEN")
+(#x10db "VERBOTENER TEMPEL (AUSGANG)")
+(#x10dc "LURKER-MASCHINE")
+(#x10dd "SPITZE DES TURMS")
+(#x10de "BLAUER-ECO-SCHALTER")
+(#x10df "UNBENUTZT")
+(#x10e0 "WÄCHTERSTATUE")
+(#x10e1 "LURKER-KANONE")
+(#x10e2 "GROTTE")
+(#x10e3 "UNBENUTZT 1")
+(#x10e4 "LURKER-BOOT")
+(#x10e5 "UNBENUTZT 2")
+(#x10e6 "ZOOMER")
+(#x10e7 "FLUT FLUT")
+(#x10e8 "UNBENUTZT")
+(#x10e9 "KRIEGER")
+(#x10ea "GEOLOGIN")
+(#x10eb "GLÜCKSSPIELER")
+(#x10ec "SCHWEBEMASCHINE")
+(#x10ed "UNBENUTZT 1")
+(#x10ee "PRECURSOR-STARTER-FALLEN")
+(#x10ef "UNBENUTZT 2")
+(#x10f0 "TOTE SCHLUCHT")
+(#x10f1 "MITTE DES PASSES")
+(#x10f2 "ENDE DES PASSES")
+(#x10f3 "WEG ZUR SPINNENHÖHLE")
+(#x10f4 "WEG ZUM SCHNEEGEBIRGE")
+(#x10f5 "BERGARBEITER")
+(#x10f6 "ROBOTERHÖHLE - KLETTERGERÜST")
+(#x10f7 "SPITZE DES KLETTERGERÜSTS")
+(#x10f8 "HAUPTHÖHLE")
+(#x10f9 "DUNKLE HÖHLE")
+(#x10fa "UNBENUTZT")
+(#x10fb "VERSTECKTE HÖHLE")
+(#x10fc "LURKER-FORT")
+(#x10fd "SCHNEEBÄLLE")
+(#x10fe "MITTE DER RÖHRE")
+(#x10ff "ENDE DER RÖHRE")
+(#x1100 "GELBER WEISE")
+(#x1101 "ROTER WEISE")
+(#x1102 "BLAUER WEISE")
+(#x1103 "ZENTRUM DER ZITADELLE")
+(#x1104 "MITTE DES BOSSKAMPFES")
+(#x1105 "ENDE DES BOSSKAMPFES")
+(#x1106 "VARIANTE 1")
+(#x1107 "VARIANTE 2")
+
+(#x1110 "ENGLISH (UK)")
+(#x1111 "PORTUGUÊS")
+(#x1112 "PORTUGUÊS (BRASIL)")
+(#x1113 "SUOMI")
+(#x1114 "SVENSKA")
+(#x1115 "DANSK")
+(#x1116 "NORSK")
+
+;; -----------------
+;; test
+
+(#x7fff
+  "RÜSTUNG"
+  )
```

Comment:
you don't need this by the way, it was just from me testing the text compiler

Reply:
Ah yeah, I figured, I can get rid of it real quick.